addons/packages/packetfence.spec

```diff
@@ -505,6 +505,8 @@ done
 %{__install} -d -m2770 $RPM_BUILD_ROOT/usr/local/pf/conf
 %{__install} -d $RPM_BUILD_ROOT/usr/local/pf/conf/radiusd
 %{__install} -d $RPM_BUILD_ROOT/usr/local/pf/conf/ssl
+%{__install} -d $RPM_BUILD_ROOT/usr/local/pf/conf/ssl/acme-challenge
+%{__install} -D -m0644 conf/ssl/acme-challenge/success.txt $RPM_BUILD_ROOT/usr/local/pf/conf/ssl/acme-challenge/success.txt
```

Comment:
It looks that success.txt is missing in the code.

Reply:
Actually forgot to remove it from the packaging since it uses a dynamic filename now